sites/default/menu_data.json

```diff
@@ -471,6 +471,14 @@
             "children" : [ ],
             "requirement" : 0,
             "acl_req" : [ "admin", "users" ]
+        },{
+
+            "label" : "User Login Report",
```

Comment:
Make this say "User Activity Report"  it's more that 'login' now

Reply:
Fixed